tensorflow_hub/resolver.py

```diff
@@ -501,7 +504,9 @@ class HttpResolverBase(Resolver):
   """Base class for HTTP-based resolvers."""
 
   def __init__(self):
-    self._context = None
+    self._context = ssl.create_default_context()
+    # Keep this function here because _set_url_context is never called
```

Comment:
Please remove the comment. `_set_url_context` is not used in the library but it's useful for supporting custom SSL certificates